apps/console/src/features/roles/components/wizard-updated/role-permissions/role-permissions.tsx

```diff
@@ -104,22 +140,40 @@ export const RolePermissionsList: FunctionComponent<RolePermissionsListProp> =
             }
         }, [ selectedAPIResourceFetchRequestError, apiResourcsListFetchRequestError ]);
 
+        /**
+         * API resources list options.
+         */
         useEffect(() => {
             const options: DropdownProps[] = [];
-    
-            apiResourcesList?.apiResources?.map((apiResource: APIResourceInterface) => {
-                if (!selectedAPIResources.find((selectedAPIResource: APIResourceInterface) => 
-                    selectedAPIResource?.id === apiResource?.id)) {
-                    options.push({
-                        key: apiResource.id,
-                        text: apiResource.name,
-                        value: apiResource.id
-                    });
-                }
-            });
+
+            if(roleAudience === RoleAudienceTypes.ORGANIZATION) {
+                // // API resources list options when role audience is "organization".
+                apiResourcesList?.apiResources?.map((apiResource: APIResourceInterface) => {
+                    if (!selectedAPIResources.find((selectedAPIResource: APIResourceInterface) => 
```

Comment:
Shall we improve formatting?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4305/commits/3b9149625c26c84e4459c2946c99c8c2d7cf4de3